lib/KCard/index.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :to="to"
+  >
+    <template #default>
+      <aside
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
```

Comment:
We should avoid burned strings in the code and instead use constants to avoid any typos and make it easier to maintain in the future if we want to change the string.

So we can maybe declare constants:

```javascript
const CardLayout = {
  VERTICAL: "vertical",
  HORIZONTAL: "horizontal",
};

const CardThumbnailDisplay = {
  NONE: "none",
  SMALL: "small",
  LARGE: "large",
};
```

Reply:
I am afraid to take this approach  because  a couple of things could go wrong  here with the` CardLayout` and `CardThumbnailDisplay` typos in the enum values (e.g., "vertival" instead of "vertical") could lead to unexpected behavior or errors.
If you accidentally use a misspelled value (e.g., CardLayout.VERTICLE), it wouldn't match any defined value in the enum

Another thing I am thinking of is that that  it only cover basic layout and thumbnail display options, it only define string values.  And If we need to associate additional data with each layout or display option this  might not be sufficient.